download_MERRA2.sh

```diff
@@ -41,6 +41,6 @@ python3 print_wget_commands.py ${year_start} ${year_end} ${month_start} ${month_
 bash retrieve_data.sh
 
 # Process files into .nc
-mkdir -p nc_files
-rm nc_files/*.nc
-csh create_forcing_nc.sh $year_start $year_end
+# mkdir -p nc_files
+# rm nc_files/*.nc
+# csh create_forcing_nc.sh $year_start $year_end
```

Comment:
Do these changes have an intended purpose?

Reply:
I commented those out for my own use; I didn't need the post processing of the downloaded files. The comment in it is a bit unclear - wget is retrieving netCDF files; is this concatenating all of the several thousand daily files into yearly files? If possible it would be good in this script to change the hard coded paths to variables set at the beginning.